.github/workflows/poc1-gar-pipeline.yaml

```diff
@@ -0,0 +1,48 @@
+name: POC1 GAR Pipeline
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - poc1-api/**
+  pull_request:
+    paths:
+      - poc1-api/**
+
+env:
+  PROJECT_ID: looker-hackathon-2023
+  GAR_LOCATION: us-central1
+  REPOSITORY: poc1-api
+  IMAGE: main
+
+jobs:
+  gar-login-build-push:
+    name: GAR Login, Build & Push
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: 'Google Auth'
+        uses: 'google-github-actions/auth@v2'
+        with:
+          credentials_json: '${{ secrets.GCP_SA_KEY }}'
+
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v1'
+        with:
+          project_id: '${{ env.PROJECT_ID }}'
+
+      - name: 'Docker Auth'
+        run: |-
+          gcloud auth configure-docker ${{ env.GAR_LOCATION }}-docker.pkg.dev
+
+      - name: 'Docker Build'
+        working-directory: poc1-api
+        run: |-
+          docker build . --file Dockerfile --tag "${{ env.GAR_LOCATION }}-docker.pkg.dev/${{ env.PROJECT_ID }}/${{ env.REPOSITORY }}/${{ env.IMAGE }}:${{ github.sha }}"
```

Comment:
Does the `docker build` happen in `poc1-api/` or in root of repo? It should happen in subdir.

Reply:
According to [the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory), the `working-directory` key on [line 42](https://github.com/IllyaMoskvin/looker-python/pull/1/commits/720c7112d7bd16766ba1682c8ae7287f85c42a68#diff-09fd6f94920907e0d590b50aa8de11ae352970d4b3b9fdc870e406d04a9f56ddR42) should cause this step to run within that directory.

Reply:
Nevermind, I just noticed the `working-directory` parameter. LGTM!